packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -187,12 +176,9 @@ class FlutterDriver {
     String dartVmServiceUrl,
     bool printCommunication = false,
     bool logCommunicationToFile = true,
-    double timeoutMultiplier = _kDefaultTimeoutMultiplier,
     int isolateNumber,
-    Duration isolateReadyTimeout,
```

Comment:
I support this PR, but it's worth understanding the context in which this timeout was added (in https://github.com/flutter/flutter/pull/16762) -- we were seeing cases where the CI container was timing out after indefinitely waiting for a future to complete, and when that happened, the driver test harness wasn't getting a chance to clean up and write out its logs, so we had no context as to what was hanging.

Reply:
We should add flushing or something in that case, to make sure we're writing the logs promptly. A crash would presumably have the same problem, and a timeout won't help there.